### ISSUE TITLE
Android PWA icon too small

### ISSUE THREAD

**Issue by chris13524** [WalletConnect/web3inbox#359]
On Android:

![Image](https://github.com/WalletConnect/web3inbox/assets/1979423/bf1c1417-465b-40a4-a1aa-e04c9dec0744)

[Context](https://walletconnect.slack.com/archives/C044SKFKELR/p1704827970632749)

**Comment by chadyj** [comment]
@chris13524 Looks good on iOS.
Is this an Android issue?

![Image](https://github.com/WalletConnect/web3inbox/assets/116099/2cca98e4-8b89-435b-81fd-d5e96739895b)

**Comment by enesozturk** [comment]
@chadyj Yeah, Might be related to Android only since the operating systems require different sizes of icons. I tried to give a variety of them but might be missing a few different sizes. Should be a short fix

**Comment by chadyj** [comment]
Icon looks pretty good on my android device


![Image](https://github.com/WalletConnect/web3inbox/assets/116099/13229c63-4f92-410a-9f3d-4bf7203356b0)

OnePlus Nord CE 2
OxygenOS 13.0
Android version 13

**Comment by chris13524** [comment]
I am:
- Pixel 6 Pro
- Android 14
- PWA via Chrome

**Comment by chris13524** [comment]
Guessing it has to do with this: https://web.dev/articles/maskable-icon

**Comment by chris13524** [comment]
Icon is still small on my device even after re-installing
